serverless/pytorch/foolwood/siammask/nuclio/function.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
 
 spec:
   description: Fast Online Object Tracking and Segmentation
-  runtime: 'python:3.6'
+  runtime: 'python:3.7'
```

Comment:
Seems like a short-term fix. Python 3.7 has been EOL for 9 months already; I'm sure Nuclio will discontinue support for it in the near future too. Why not try the most recent supported version of Python instead?

Reply:
Also, this isn't the only function that uses the `python:3.6` runtime. What about these other ones?

```console
$ rg -l python:3.6 serverless/
serverless/pytorch/saic-vul/fbrs/nuclio/function.yaml
serverless/pytorch/foolwood/siammask/nuclio/function-gpu.yaml
serverless/pytorch/foolwood/siammask/nuclio/function.yaml
serverless/tensorflow/faster_rcnn_inception_v2_coco/nuclio/function-gpu.yaml
serverless/tensorflow/matterport/mask_rcnn/nuclio/function-gpu.yaml
serverless/tensorflow/faster_rcnn_inception_v2_coco/nuclio/function.yaml
serverless/tensorflow/matterport/mask_rcnn/nuclio/function.yaml
```